nftest/common.py

```diff
@@ -27,6 +27,14 @@ def remove_nextflow_logs() -> None:
         else:
             os.remove(file)
 
+def resolve_single_path(path:str) -> Path:
+    """ Resolve wildcards in path and ensure only a single path is identified """
+    expanded_paths = glob.glob(path)
+    assert len(expanded_paths) == 1, \
```

Comment:
Be wary of relying on `assert`. Context [here](https://discuss.python.org/t/stop-ignoring-asserts-when-running-in-optimized-mode/13132).